src/idlcxx/src/generator.c

```diff
@@ -278,16 +278,15 @@ static void copy(
   memmove(dest+off, src, cnt);
 }
 
-int get_cpp11_fully_scoped_name(
+int get_cpp11_fully_scoped_name_seps(
   char *str, size_t size, const void *node, void *user_data)
 {
-  const char *name, *sep = "::";
+  const char *name, *sep = user_data;
```

Comment:
Probably best to change `void *user_data` to `const char *sep`. Don't think the function is used for `IDL_PRINTA` itself, so no point in keeping it generic. Also, probably best to change it to `static` because it's only used within this one file.

Reply:
You are correct in this, it will be changed.